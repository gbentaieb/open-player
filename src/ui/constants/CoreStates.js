export const PLAYING = 'PLAYING';
export const SEEKING = 'SEEKING';
export const BUFFERING = 'BUFFERING';
export const STOPPED = 'STOPPED';
export const LOADING = 'LOADING';
export const LOADED = 'LOADED';
export const ENDED = 'ENDED';
export const PAUSED = 'PAUSED';

export const CORE_STATES = [PLAYING, SEEKING, BUFFERING, STOPPED, LOADING, LOADED, ENDED, ENDED];
