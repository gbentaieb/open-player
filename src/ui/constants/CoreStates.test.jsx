import * as coreStates from './CoreStates';

describe('CoreStates > Snapshot', () => {
  test('CoreStates > Snapshot', () => {
    expect(coreStates).toMatchSnapshot();
  });
});
