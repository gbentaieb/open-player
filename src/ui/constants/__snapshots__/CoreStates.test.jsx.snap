// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoreStates > Snapshot CoreStates > Snapshot 1`] = `
Object {
  "BUFFERING": "BUFFERING",
  "CORE_STATES": Array [
    "PLAYING",
    "SEEKING",
    "BUFFERING",
    "STOPPED",
    "LOADING",
    "LOADED",
    "ENDED",
    "ENDED",
  ],
  "ENDED": "ENDED",
  "LOADED": "LOADED",
  "LOADING": "LOADING",
  "PAUSED": "PAUSED",
  "PLAYING": "PLAYING",
  "SEEKING": "SEEKING",
  "STOPPED": "STOPPED",
}
`;
